tests/e2e/alltests.js

```diff
@@ -64,5 +63,14 @@ test('Test update user role', async t => {
         .contains('Roles updated successfully');
 });
 
+test('Test search by administrator', async t => {
+    await t
+        .click('#admin-event-log-link')
+        .typeText('#admin-id', TEST_CAFE_USER_ID)
+        .pressKey('enter')

```

Comment:
Just wanted to point out that I used the `pressKey('enter')` action instead of clicking the search button.